lib/testscript_engine/testscript_runnable.rb

```diff
@@ -77,16 +79,16 @@ def preprocess
 
   def setup
     return info(:no_setup) unless script.setup
-    handle_actions(script.setup.action, true)
+    handle_actions(script.setup.action, :setup)
```

Comment:
This is a good way to detail the configuration of fail on error in different phases of testing